vocabularies/Common.json

```diff
@@ -1289,6 +1289,95 @@
       "@Common.Experimental": true,
       "@Core.IsURL": true,
       "@Core.Description": "Base URL for WebSocket connections"
+    },
+    "GetRecommendations": [
+      {
+        "$Kind": "Function",
+        "$IsBound": true,
+        "@Common.Experimental": true,
+        "@Core.IsSignature": true,
+        "@Core.Description": "Computes AI-based recommendations for the bound entity and its related entities",
+        "@Core.LongDescription": "The function SHOULD compute the recommendations asynchronously, see [this diagram](../docs/recommendations.md).",
+        "$Parameter": [
+          { "$Name": "Entity", "$Type": "Edm.EntityType" },
+          {
+            "$Name": "Targets",
+            "$Type": "Common.RecommendationsTargetsType",
+            "@Core.Description": "Entities, properties or action/function parameters for which recommendations shall be retrieved",
+            "@Core.LongDescription": "Implementations of this function MAY omit this parameter.\n            If this parameter is omitted (by the implementation or in the invocation), all recommendations are retrieved.",
+            "@Core.OptionalParameter": {}
+          }
+        ],
+        "$ReturnType": {
+          "$Collection": true,
+          "$Type": "Common.PropertyRecommendationType",
+          "@Core.Description": "A collection of recommendations that differ in their `AIRecmddFldURL` property"
+        }
+      }
+    ],
+    "RecommendationsTargetsType": {
+      "$Kind": "ComplexType",
+      "@Common.Experimental": true,
+      "URLs": {
+        "$Collection": true,
+        "@Core.IsURL": true,
+        "@Core.Description": "URLs (relative to the request URL) that address entities, properties or action/function parameters for which recommendations shall be retrieved",
+        "@Core.LongDescription": "The request `SalesOrder('A')/ns.GetRecommendations?Targets={\"URLs\":[\"\",\"Items(10)\"]}`\n            retrieves recommendations for the entities `SalesOrder('A')` and `SalesOrder('A')/Items(10)`."
+      }
+    },
+    "PropertyRecommendationType": {
+      "$Kind": "ComplexType",
+      "@Common.Experimental": true,
+      "AIRecmddFldURL": {
```

Comment:
Can we use non-abbreviated names proposed by Costa in the last mail like "AIRecommendedFieldURL"?